src/toolbars/ControlToolBar.cpp

```diff
@@ -517,6 +517,35 @@ void ControlToolBar::SetStop()
    EnableDisableButtons();
 }
 
+#include "../SpectrumTransformer.h"
+namespace
+{
+
+auto BandPasser(size_t windowSize, double rate, double freqLo, double freqHi)
+   -> SpectrumTransformer::WindowProcessor
+{
+   auto bin = rate / windowSize;
+   size_t spectrumSize = (1 + windowSize / 2);
+   return [
+      spectrumSize,
+      mBinLo = size_t(freqLo < 0 ? 0 : floor(freqLo / bin)),
+      mBinHi = size_t(freqHi < 0 ? spectrumSize : ceil(freqHi / bin))
```

Comment:
Not using `round` for both high and low bins here, in order to get the bin range as wide as possible?